store/evm_aux/receipts.go

```diff
@@ -0,0 +1,198 @@
+package evmaux
```

Comment:
Everything in the EvmAuxStore is related to receipts, so I don't understand why this code here is in a separate file from the rest of it.

Reply:
There are three things stored EvmAuxStore. 
- BloomFilter
- TxHashList
- Receipts

So the receipts code go to `receipts.go` but the rest will be in `store.go`

Reply:
The receipts code is what writes all those things, yet the DB keys and getters are in `store.go`.

Reply:
ok moved everything to `store.go`